docs/releases/1.26-NOTES.md

```diff
@@ -82,6 +82,13 @@ CNIs, use the "cni" networking option instead.
 
 * All legacy addons are deprecated in favor of managed addons, including the [metrics server addon](https://github.com/kubernetes/kops/tree/master/addons/metrics-server) and the [autoscaler addon](https://github.com/kubernetes/kops/tree/master/addons/cluster-autoscaler).
 
+# Full change list since 1.25.0 release
+
+
+## 1.26.0-beta.2 to 1.26.0-beta.3
+
+* Add unreachable route for pod IP on deletion option in Cilium [@anthonyhaussman](https://github.com/anthonyhaussman) [#15050](https://github.com/kubernetes/kops/pull/15050)
```

Comment:
This bit is autogenerated. You can add a section `Other changes of note` towards the end. See https://kops.sigs.k8s.io/releases/1.23-notes/#other-changes-of-note

Reply:
Just in the 1.26 release notes, not 1.27 also.

Reply:
I correct this :ok_hand:

Reply:
I hope it's okay now @olemarkus @hakman :slightly_smiling_face: